library/lab.inc.php

```diff
@@ -84,15 +84,15 @@ function getSelfPay($pid)
  */
 function getNPI($prov_id)
 {
-    $sql = "SELECT npi, upin FROM users WHERE id = ?";
+    $sql = "SELECT `npi`, `upin` FROM `users` WHERE `id` = ?";
     $res = sqlQuery($sql, array($prov_id));
     return array($res['npi'], $res['upin']);
 }
 
 /**
  * @return array
  */
-function getProcedureProvider($prov_id)
+function getProcedureProvider($prov_id): array
```

Comment:
sqlFetchArray can return an array or false.  Should probably set line 101 to be $labs = sqlFetchArray($res) ?? []; or change the return type of this function.